src/OrbitBase/include/OrbitBase/GetProcessIds.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_GET_PROCESS_IDS_H
+#define ORBIT_BASE_GET_PROCESS_IDS_H
+
+#include <sys/types.h>
+
+#include <vector>
+
+namespace orbit_base {
+
+#if defined(__linux)
+// Get the process ids of all currently running processes.
+std::vector<pid_t> GetAllPids();
```

Comment:
Get(All)(Running)ProcessIds

I think we follow ThreadId ProcessId naming conventions elsewhere in OrbitBase